### ISSUE TITLE
Custom Attributes vs Incomplete Tags

### ISSUE THREAD

**Issue by andrewnordlund** [andrewnordlund/nordValidator#19]
In the following code:
```
<p class="blah xyz">Blah</p>
<p class="blah" xyz>Blah blah</p>
<p class=blah xyz>Blah blah blah</p>
```
Line 1 is totally valid.
Line 2 is not valid, but is not a WCAG 4.1.1 - Parsing issue.
Line 3 is not valid, but _may be_ a WCAG 4.1.1 - Parsing issue.  Is `xyz` a custom attribute, or another CSS class name? If it's the former, WCAG is not concerned.  If it's the latter, then the tag isn't "complete" and, therefore is a WCAG issue.

At this point, the add-on won't distinguish between the errors returned for lines 2 and 3.  But it should.

Custom attributes that have values are easy to look for.  They have an equals sign.  Stand-alone custom attributes, however, aren't so easy. 

A solution should be found.
